translations/messages+intl-icu.de.yml

```
@@ -1029,6 +1029,7 @@ error.copy.mapfile: "Die Einzeichnung der Stellungnahme {externId} konnte nicht
 error.changes.not.saved: "Beim Speichern der Änderungen ist ein Fehler aufgetreten."
 error.date.authored: "Bitte geben Sie ein Verfassungsdatum ein."
 error.in.fields: "Bitte korrigieren Sie Ihre Eingabe in folgenden Feldern: "
+error.phone.required: "Bitte geben sie eine gültige Telefonnummer ein."
```

```suggestion
error.phone.invalid: "Bitte geben sie eine gültige Telefonnummer ein."
```
I think the key should transport the info that the phone number is invalid

---

here it has to be required name cause the other errors have the same name and the focus is that its required. the focus is not on invalid

---

But we want to have generic translations hence the naming of the translation key should not be dependent on a specific area in the code. Or the wording of the key must be adjusted imho

---

done https://github.com/demos-europe/demosplan-core/pull/2156/commits/4e5a64bd4000ddeb2b234fbe8925ab94b191600a